packages/core/src/format/format.ts

```diff
@@ -41,7 +41,7 @@ const CODECLIMATE_SEVERITY_MAPPING = {
   warn: 'minor',
 };
 
-const MAX_SUGGEST = 5;
+const MAX_SUGGEST = parseInt(env.REDOCLY_MAX_SUGGESTIONS || '7', 10);
```

Comment:
Why not leaving it 5? The users have the ability to customise it the way they want.

```suggestion
const MAX_SUGGEST = +env.REDOCLY_MAX_SUGGESTIONS ?? 5;
```